Backend/lib/special-zone/src/Lib/Tabular/SpecialLocationGeom.hs

```diff
@@ -51,5 +51,6 @@ instance ToTType SpecialLocationGeomT Domain.SpecialLocation where
       { id = getId id,
```

Comment:
looks like `SpecialLocationGeomT` represents the same table as `SpecialLocationT`
`SpecialLocationGeomT sql=special_location`
so, lets add `merchantId` here too